src/operation/count/mod.rs

```diff
@@ -47,8 +47,10 @@ impl Operation for Count {
     const NAME: &'static str = "count";
 
     fn build(&mut self, description: &StreamDescription) -> Result<Command> {
+        let mut name = Self::NAME.to_string();
         let mut body = match description.max_wire_version {
             Some(v) if v >= SERVER_4_9_0_WIRE_VERSION => {
+                name = "aggregate".to_string();
```

Comment:
This has actually been broken for a while; the test that should have caught it was passing because the `command_name` field didn't have a camelCase rename and the unparsed `commandName` field was being ignored as an unknown field.